src/Tizen.TV.UIControls.Forms/Renderer/GridViewRenderer.cs

```diff
@@ -0,0 +1,168 @@
+                    ItemAlignmentY = Element.ItemVerticalAlignment,
+                    Style = Element.ThemeStyle,
+                };
+                _genGrid.ItemSelected += OnItemSelected;
+                _genGrid.ItemFocused += OnItemFocused;
+                _genGrid.ItemUnfocused += OnItemUnfocused;
+                
+                if (Element.ItemsSource != null)
+                {
+                    UpdateItemsSource();
+                }
+                SetNativeControl(_genGrid);
+            }
+            base.OnElementChanged(e);
+        }
+
+        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == GridView.ItemWidthProperty.PropertyName)
+            {
+                Control.ItemWidth = Element.ItemWidth;
+            }
+            else if (e.PropertyName == GridView.ItemHeightProperty.PropertyName)
+            {
+                Control.ItemHeight = Element.ItemHeight;
+            }
+            else if (e.PropertyName == GridView.ThemeStyleProperty.PropertyName)
+            {
+                Control.Style = Element.ThemeStyle;
+            }
+            else if (e.PropertyName == GridView.ItemsSourceProperty.PropertyName)
+            {
+                UpdateItemsSource();
+            }
+            base.OnElementPropertyChanged(sender, e);
+        }
+
+        void OnItemSelected(object sender, GenGridItemEventArgs e)
+        {
+            GengridItemContext context = e.Item.Data as GengridItemContext;
+            Element.SelectedItem = context.Data;
+        }
+
+        void OnItemUnfocused(object sender, GenGridItemEventArgs e)
+        {
+            GengridItemContext context = e.Item.Data as GengridItemContext;
+            Element.SendItemFocused(new GridViewFocusedEventArgs(context.Data, context.RealizedView, false));
+        }
+
+        void OnItemFocused(object sender, GenGridItemEventArgs e)
+        {
+            GengridItemContext context = e.Item.Data as GengridItemContext;
+            Element.SendItemFocused(new GridViewFocusedEventArgs(context.Data, context.RealizedView, true));
+        }
+
+        void UpdateItemsSource()
+        {
+            _genGrid.Clear();
+            itemContexts.Clear();
+
+            if (Element.ItemsSource is INotifyCollectionChanged _collection)
+            {
+                _collection.CollectionChanged -= OnCollectionChanged;
+            }
+
+            if (Element.ItemsSource == null)
+            {
+                return;
+            }
+
+            foreach (var item in Element.ItemsSource)
+            {
+                View realview = ContentCreator.CreateContent(Element.ItemTemplate, item, Element.ItemHeight, Element.ItemWidth);
+                realview.BindingContext = item;
+                var context = new GengridItemContext
+                {
+                    Data = item,
+                    RealizedView = realview as View,

```

Comment:
`realview` already `View` type
```suggestion
                    RealizedView = realview,

```

Reply:
I will fix it. Thank you.